tests/components/zwave_mqtt/test_sensor.py

```diff
@@ -0,0 +1,76 @@
+"""Test Z-Wave Sensors."""
+from homeassistant.components.sensor import (
+    DEVICE_CLASS_HUMIDITY,
+    DEVICE_CLASS_POWER,
+    DEVICE_CLASS_PRESSURE,
+    DOMAIN as SENSOR_DOMAIN,
+)
+from homeassistant.components.zwave_mqtt.const import DOMAIN
+from homeassistant.const import ATTR_DEVICE_CLASS
+
+from .common import setup_zwave
+
+
+async def test_sensor(hass, generic_data):
+    """Test setting up config entry."""
+    await setup_zwave(hass, fixture=generic_data)
+
+    # Test standard sensor
+    state = hass.states.get("sensor.smart_plug_electric_v")
+    assert state is not None
+    assert state.state == "123.9"
+    assert state.attributes["unit_of_measurement"] == "V"
+
+    # Test device classes
+    state = hass.states.get("sensor.trisensor_relative_humidity")
+    assert state.attributes[ATTR_DEVICE_CLASS] == DEVICE_CLASS_HUMIDITY
+    state = hass.states.get("sensor.trisensor_pressure")
+    assert state.attributes[ATTR_DEVICE_CLASS] == DEVICE_CLASS_PRESSURE
+    state = hass.states.get("sensor.trisensor_fake_power")
+    assert state.attributes[ATTR_DEVICE_CLASS] == DEVICE_CLASS_POWER
+    state = hass.states.get("sensor.trisensor_fake_energy")
+    assert state.attributes[ATTR_DEVICE_CLASS] == DEVICE_CLASS_POWER
+    state = hass.states.get("sensor.trisensor_fake_electric")
+    assert state.attributes[ATTR_DEVICE_CLASS] == DEVICE_CLASS_POWER
+
+    # Test ZWaveListSensor disabled by default
+    registry = await hass.helpers.entity_registry.async_get_registry()
+    entity_id = "sensor.water_sensor_6_instance_1_water"
+    state = hass.states.get(entity_id)
+    assert state is None
+
+    entry = registry.async_get(entity_id)
+    assert entry
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5462445)